azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -30,6 +34,16 @@
   QUEUED(110),
   FAILED_SUCCEEDED(120),
   CANCELLED(130);
+  // status is TINYINT and in H2 DB the possible values are: -128 to 127
```

Comment:
So far there is no need to create separate sql DDLs for h2 yet. 
I think it is simpler to change the schema for both DBs.